app/src/main/java/one/mixin/android/ui/sticker/AlbumAdapter.kt

```diff
@@ -60,7 +67,9 @@ class AlbumHolder(
                             .showNow(fragmentManager, StickerAlbumBottomSheetFragment.TAG)
                     }
                 }
-                adapter.submitList(album.stickers)
+                viewModel.observeSystemStickersByAlbumId(album.albumId).observe(lifecycleOwner) { stickers ->
```

Comment:
Maybe it should use [switchMap](https://developer.android.com/reference/androidx/lifecycle/Transformations#switchMap(androidx.lifecycle.LiveData%3CX%3E,%20androidx.arch.core.util.Function%3CX,%20androidx.lifecycle.LiveData%3CY%3E%3E))